helpers/formValidation.tsx

```diff
@@ -1,107 +1,121 @@
 import * as Yup from 'yup'
 
-const TEXT_MIN = 2
-const TEXT_MAX = 64
-const PASSWORD_MIN = 6
+enum Constants {
+  Required = 'Required',
+  TextMin = 2,
+  TextMax = 64,
+  PasswordMin = 6,
+  LeadingTrailing = 'Leading and trailing value must be alphanumeric character',
+  NumbersOnly = 'Numbers only',
+  PositiveNumbers = 'Positive numbers only',
+  AlphaNumChars = 'Must be alphanumerics characters',
+  MustBeLowercase = 'Must be a lowercase string'
+}
+
 const REGEX_ALPHANUMERICS_AND_SPACE = /^[a-zA-Z0-9_\s]*$/
+const MustAtLeastTextMinChars = `Must be at least ${Constants.TextMin} characters`
```

Comment:
This can't go into the enum because it doesn't allow computed values such as `${}`

Reply:
The convention for constants is to user uppercase with words separated by underscore, please rename the variable names for these constants
https://google.github.io/styleguide/jsguide.html#naming-constant-names

Reply:
Actually since we have these strings that have computed values, I suggest we have an `ERROR_MESSAGES` object instead of the `ErrorMessages` I suggested in my other comment. This way we can just add these constants to the ERROR_MESSAGES object, which will improve readability

Reply:
I agree with you. I'll apply your comments soon.